tests/dummy_qrc/zurich.py

```diff
@@ -133,6 +139,8 @@ def create(runcard_path=RUNCARD):
     # create qubit objects from runcard
     runcard = load_runcard(runcard_path)
     qubits, pairs = load_qubits(runcard)
+    couplers = load_couplers(runcard)
```

Comment:
We could merge these two in a single function like
```suggestion
    qubits, pairs, couplers = load_qubits(runcard)
```
where `couplers == None` if they do not exist in the runcard. This would also include the `register_gates`, as it was before, so it is a bit simpler for users that have to write `create` functions.

Reply:
Maybe this can be solved if we do https://github.com/qiboteam/qibolab/pull/508#discussion_r1335466568 and assign qubits and coupler in that function before going to the `register_gates` one

Reply:
We don't need to make any implementation change for this, just merge what these three functions are already doing to a single one. But in the end it is a matter of preference: either a single big function or three smaller. Usually smaller functions are better and more flexible, I was just thinking from a user perspective it may be simpler to have only one. But I am fine either way, so feel free to ignore this.

We just need to update the documentation (tutorial on how to create platforms) with the final decision. And of course also the platforms in qibolab_platforms_qrc, but this I will do later.

Reply:
I went with the single one, was easier to create platforms like that.